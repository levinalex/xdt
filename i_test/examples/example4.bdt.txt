01380000020
014810000084
0189100030456100
017910324022010
0129105001
01091062
01380000022
014810000107
014921010/93
014921302/94
01096002
02596012402201024022010
017960215020000
01380006100
014810000153
01030001
0143101Anton
0153102Anton#
017310301121960
021310628779 Bremen
0293107Schwaneweder Str. 3b
01031083
01031101
01380006200
014810000249
01030001
0136311Herr
0326310Dr.Dieter Braas-Kastien
0286312Blumenthalerstr. 29
014631328790
0206314Schwanewede
019631604209/3536
01663191484057
0206330Briefanrede
0506331Sehr geehrter Herr Kollege Braas-Kastien,
01380000023
014810000056
017920200000649
0129203001
01380000021
014810000027
