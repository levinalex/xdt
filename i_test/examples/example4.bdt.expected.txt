{
  "patients": [{
      "xref": "1",
      "first_name": "Anton#",
      "last_name": "Anton",
      "gender": "male",
      "zip": "28779",
      "city": "Bremen",
      "street": "Schwaneweder Str. 3b",
      "born_on": "1960-12-01"
    }]
}